méli-mélo/2021-05-conjunction/index.html

```diff
@@ -100,10 +102,79 @@ <h4>Always stacked (mobile first, no side-by-side)</h4>
 	&lt;li class=&quot;cnjnctn-col&quot;&gt;
 		This is content for part B
 	&lt;/li&gt;
-&lt;/li&gt;</code>
+&lt;/li&gt;
+&lt;/ul&gt;</code>
   </pre>
   </details>
 
+<h3>No border on always stacked <span class="label label-info">New</span></h3>
+<p>This can be used when a border or design (e.g. zebra striping) has already defined the border of the content.</p>
+<h4>Always stacked (mobile first, no side-by-side)</h4>
+<div class="panel panel-default">
```

Comment:
This structure need to be a sectioning element in order to have the "details/summary" content for the source code associated with the heading 4

Reply:
@duboisp  I have made the amendments, I have not yet created the test page for the options where the brdr-0 does not work, I can create that tomorrow.

Reply:
@Christopher-O I added the test page, we will be able to merge your PR now and include into our next deployment in AEM